File-Formats/DocIO/Working-with-Charts/Format-Axis.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Modify the appearance of axes in chart | Syncfusion
+description: Learn how to modify the appearance of axes in a chart in a Word document using the Syncfusion .NET Core Word (DocIO) library without Microsoft Word.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Chart Axis
+
+Charts typically have two axes that are used to measure and categorize data
+-  Horizontal axis (also known as category axis or x axis).
+-  Vertical axis (also known as value axis or y axis).
+
+Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library you can customize the **axis** in the chart.
+
+## Set the Axis Title
+
+The following code snippet illustrates how to set the title in axis.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Set the horizontal (category) axis title.
+chart.PrimaryCategoryAxis.Title = "Products";
+//Set the Vertical (value) axis title.
+chart.PrimaryValueAxis.Title = "In Dollars";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Border
+
+The following code snippet illustrates how to customize the border of Horizontal and vertical category axis.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Customize the horizontal category axis.
+chart.PrimaryCategoryAxis.Border.LinePattern = OfficeChartLinePattern.CircleDot;
+chart.PrimaryCategoryAxis.Border.LineColor = Syncfusion.Drawing.Color.Blue;
+chart.PrimaryCategoryAxis.Border.LineWeight = OfficeChartLineWeight.Hairline;
+
+//Customize the vertical category axis.
+chart.PrimaryValueAxis.Border.LinePattern = OfficeChartLinePattern.CircleDot;
+chart.PrimaryValueAxis.Border.LineColor = Syncfusion.Drawing.Color.Blue;
+chart.PrimaryValueAxis.Border.LineWeight = OfficeChartLineWeight.Hairline;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Font
+
+The following code snippet illustrates how to customize the border of Horizontal and vertical category axis.
```

Comment:
The following code sample illustrates how to customize the border of Horizontal and vertical category axis.